src/index.js

```diff
@@ -1,3 +1,3 @@
-export {default as sankey} from "./sankey.js";
-export {center as sankeyCenter, left as sankeyLeft, right as sankeyRight, justify as sankeyJustify} from "./align.js";
-export {default as sankeyLinkHorizontal} from "./sankeyLinkHorizontal.js";
+export {sankeyTop, sankeyRight, sankeyBottom, sankeyLeft} from "./sankey.js";
+export {center as sankeyAlignCenter, left as sankeyAlignLeft, right as sankeyAlignRight, justify as sankeyAlignJustify} from "./align.js";
```

Comment:
This is partly for disambiguation and partly to maintain consistency with the other alignment-related variables.

The concepts of `left` and `right` can now apply *either* to the direction of the chart *or* to the alignment of the chart within the bounding box.  To reduce confusion we therefore keep separate `sankeyLeft` and `sankeyAlignLeft` variables.

Since `center` referred to an alignment within the bounding box, we rename it to `sankeyAlignCenter` for consistency.

This does make me wonder whether `sankeyLeft`, `sankeyRight`, etc -- the rendering directions -- should be renamed to `sankeyFlowLeft` and `sankeyFlowRight` or similar; that might further help readers to understand the difference.  What do you reckon?

Reply:
Got it. Thanks for the context! It makes sense as-is.